Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -2958,6 +2958,38 @@ public void Editor_CanRestartEditorThroughReflection()
         EditorHelpers.RestartEditorAndRecompileScripts(dryRun: true);
     }
 
+    [Test]
+    [Category("Editor")]
+    public void Editor_InputControlPicker_TouchscreenPickerContainsSingleAndMultiTouchControls()
+    {
+        var dropdown = new StubInputControlPickerDropdown(new InputControlPickerState(), _ => {});
+        var root = dropdown.BuildRoot();
+
+        Assert.That(() =>
+        {
+            var touchscreen = root.children.FirstOrDefault(c => c.name == "Touchscreen");
+            if (touchscreen == null)
+                return false;
+
+            return touchscreen.children.Any(c => c.name == "Press (Single touch)") &&
```

Comment:
Code looks clean and readable to me. However I have one question related to the general concept here, what is the difference with respect to single press touch and multi-touch. Is single press touch a trigger mapping only to a press with a single touch point and multi-touch is anything more than a single touch point for the press, e.g. N > 1, where N = 2, 3, 4....

Reply:
Couldn't find any explanation in the docs around this other than "Using touch with Actions" under https://docs.unity3d.com/Packages/com.unity.inputsystem@1.1/manual/Touch.html either so maybe we are missing something there or did I only fail to find it? Does multi-touch here correspond to wild-card touches? Does this enforce pass-through as suggested in the manual?

Reply:
> Code looks clean and readable to me. However I have one question related to the general concept here, what is the difference with respect to single press touch and multi-touch. Is single press touch a trigger mapping only to a press with a single touch point and multi-touch is anything more than a single touch point for the press, e.g. N > 1, where N = 2, 3, 4....

Single press maps to primary touch, which is just touch0, and multi-touch maps to any touch and so will also work for 1 touch. The difference is that primary touch can "migrate" to a different finger if the finger that was considered the first touch is released, while another is still held down.

Reply:
> Couldn't find any explanation in the docs around this other than "Using touch with Actions" under https://docs.unity3d.com/Packages/com.unity.inputsystem@1.1/manual/Touch.html either so maybe we are missing something there or did I only fail to find it? Does multi-touch here correspond to wild-card touches? Does this enforce pass-through as suggested in the manual?

Yes, multi-touch corresponds to wild-card touches. I didn't make this force passthrough because there could be other bindings on the action that the user doesn't want to be passthrough. Not sure what the best solution is there. Filling in touch*/Press for the user sort of solves only half the problem.